lib/gather.js

```diff
@@ -0,0 +1,23 @@
+const fetch = require('node-fetch')
+const fs = require('fs')
+
+function getRepository() {
+  if (process.env.REPOSITORY_URL) {
+    return process.env.REPOSITORY_URL.replace('git@github.com:', '')
```

Comment:
Use the git remote ; then it works everywhere, Netlify , Travis, etc

https://www.npmjs.com/search?q=git+remote